libs/feature/search/src/lib/results-table/results-table.component.html

```diff
@@ -0,0 +1,135 @@
+<div>
+  <div class="flex records-information">
```

Comment:
Out of curiosity, the css file is empty, but the `records-information` looks like a project specific class, where does it come from? I couldn't find it as a selector in the unit tests either, although I'd much rather use the `data-test` attribute selector.

Reply:
it's used in the Cypress E2E tests; I've changed it to use a `data-cy` attribute which is a standard practice and less prone to confusion